src/program.js

```diff
@@ -258,55 +217,49 @@ export default class Program {
 
   // Alternate constructor
   // Create a program from vertex and fragment shader node ids
-  static async fromShaderIds(...args) {
-    const opt = Program._getOptions({}, ...args);
-    const vertexShader = document.getElementById(opt.vs).innerHTML;
-    const fragmentShader = document.getElementById(opt.fs).innerHTML;
-    return new Program(opt.app, vertexShader, fragmentShader);
+  static fromHTMLTemplates(app, vs, fs) {
```

Comment:
I'm starting to think that with ES6 template strings (that support multi line strings), this explicit support for getting strings from HTML templates seems increasingly old school. If an app really wants to do it, it is just one line of code, they can implement it themselves.

I think we should remove this "constructor" and update all examples by moving the shader text from HTML file to template strings in the javascript.

  const fragmentShader = `
     ...
  `;